.eslintrc

```diff
@@ -15,5 +15,6 @@
     "@typescript-eslint/explicit-function-return-type": "off",
     "@typescript-eslint/no-empty-function": "off",
     "@typescript-eslint/no-explicit-any": "off",
+    "@typescript-eslint/no-non-null-assertion": "off"
```

Comment:
I feel like it is a good tool when used correctly, and I trust this team to use it correctly. That said, I am totally OK with leaving it on and just configuring it in the `run-regression.ts` file. @mint-thompson, what do you think?

Reply:
You convinced me. I turned it back on in eslint and just disabled it in that one file (see 2700e60). If we find that we need to use non-null assertions in lots of other places, then we can consider turning it off. Otherwise it's better to be more explicit about turning it off so that reviewers notice and pay attention to ensure that it really is safe to use.

Reply:
Oh, okay! I was going to say that I agreed that I trusted this team to use it correctly, but okay, works for me!

Reply:
I just want to make it _real easy_ for you to hit that "Approve" button. ;-)